sbt/src/main/scala/laika/sbt/LaikaPlugin.scala

```diff
@@ -37,200 +37,196 @@ import org.apache.fop.apps.FopFactory
 import sbt.Keys._
 import sbt._
 
-object LaikaSbtPlugin extends Plugin {
+object LaikaPlugin extends AutoPlugin {
+
+  val requirements = plugins.JvmPlugin
+  override val trigger = noTrigger
+
+  object autoImport {
```

Comment:
I'm not sure whether this is a good idea, because some "generic" identifiers like `generate` and `site` end up automatically imported in scope. I'll poke some sbt experts about what they think.